README.md

```diff
@@ -576,6 +576,36 @@ database {
 
 By default batch inserts will be processed in blocks of 2000. To modify this value add the field `insert-batch-size` to the `database` stanza.
 
+Cromwell metadata (paths to job results, start and end times, etc.) grow at a significantly faster rate than operations
+data (running workflows, previous call cache results, etc.). To use a separate database for operations and/or metadata,
```

Comment:
the real split is that the metadata event store contains information that will find its way to users. the other db stuff is for internal purposes